modules/javafx.media/src/main/legal/gstreamer.md

```diff
@@ -26,7 +26,14 @@ code is here:
    rt/modules/javafx.media/src/main/native/gstreamer/gstreamer-lite/
 ```
 
-### LGPL 2.1
+### LICENSE
+```
+There are multiple licenses that apply to different parts of GStreamer. The
+overarching license is LGPL 2.1, however parts are covered by BSD-style
+licenses.
+```
+
+#### LGPL 2.1
```

Comment:
Minor: to match the other sections and also what we did for WebKit, you might change this to `LGPL 2.1 License`